components/prize_pool/commons/prize_pool.lua

```diff
@@ -397,7 +397,7 @@ function PrizePool:create()
 	self.options = self:_readConfig(self.args)
 	self.prizes = self:_readPrizes(self.args)
 	self.placements = self:_readPlacements(self.args)
-	self.placements = Import.run(self.placements, self.args)
+	self.placements = Import.run(self)
```

Comment:
Wouldn't this break it for all manual tables?

Reply:
manual input works fine:
![Screenshot 2022-10-20 15 47 03](https://user-images.githubusercontent.com/75081997/196966404-5f3e0bcc-1143-411f-8ed1-1fb0f084eb67.png)
![Screenshot 2022-10-20 15 47 11](https://user-images.githubusercontent.com/75081997/196966420-c60094a3-3001-4509-9af6-790efcce1916.png)